### DISCUSSION TITLE
Error: Got unexpected extra arguments (apps config.py constants.py data dev.sh main.py migrations open_webui requirements.txt start.sh start_windows.bat static test utils)

### DISCUSSION THREAD

**Discussion by Barryhuan** [open-webui/open-webui#4084] [General]
I am using node and Python to build open webui, and after the final bash start.sh command, I encountered this error:
Loading WEBUI_SECRET_KEY from file, not provided as an environment variable.
Loading WEBUI_SECRET_KEY from .webui_secret_key
Usage: uvicorn [OPTIONS] APP
Try 'uvicorn --help' for help.

Error: Got unexpected extra arguments (apps config.py constants.py data dev.sh main.py migrations open_webui requirements.txt start.sh start_windows.bat static test utils)

node version：v20.15.1Python 3.12.4

**Comment by Barryhuan** [comment]
Do we need to install Docker?

**Reply by JaweriaAsif** [reply]
did you get a solution?

**Reply by Barryhuan** [reply]
I couldn't find it, but in the end, I still installed Docker

**Reply by ozer0306** [reply]
Did you get a solution?

**Reply by Guaderxx** [reply]
Do you guys still have this question?
I have the same question and I run the command inside the script, like `uvicorn open_webui.main:app` , and it worked.

**Reply by weilun0510** [reply]
Updated the dev.sh file: `uvicorn open_webui.main:app --port 8080`, then it worked.

[a comment by an automated account is not shown]
